opentelemetry-sdk/pyproject.toml

```diff
@@ -41,6 +41,14 @@ sdk = "opentelemetry.sdk.environment_variables"
 [project.entry-points.opentelemetry_id_generator]
 random = "opentelemetry.sdk.trace.id_generator:RandomIdGenerator"
 
+[project.entry-points.opentelemetry_traces_sampler]
+always_on = "opentelemetry.sdk.trace.sampling:_AlwaysOn"
+always_off = "opentelemetry.sdk.trace.sampling:_AlwaysOff"
+parentbased_always_on = "opentelemetry.sdk.trace.sampling:_DefaultOn"
```

Comment:
Can we make the entry point name consistent with the class name?

Here we have `parentbased_always_on` associated with `_DefaultOn`.